src/drivers/pwm_esc/Kconfig

```diff
@@ -6,7 +6,7 @@ menuconfig DRIVERS_PWM_ESC
 
 menuconfig USER_PWM_ESC
 	bool "pwm_esc running as userspace module"
-	default y
+	default n
```

Comment:
Is there some reason this is now put back to kernel space ?

Reply:
trying to save some CPU, it seems to be really heavy in user space. even though it is quite slow cycle (4 ms / cycle), it does a bunch of ioctls on every cycle and also runs in orb_callback thread, which also raises CPU consumption